mac/mac.json

```diff
@@ -1,23 +1,18 @@
 {
   "name": "libmac",
-  "no-autogen": true,
-  "rm-configure": true,
-  "make-install-args": [
-    "prefix=/app"
+  "buildsystem": "simple",
+  "build-commands": [
+    "make -f Source/Projects/NonWindows/Makefile prefix=/app all install"
```

Comment:
This is fine in that case. Just add `-j $FLATPAK_BUILDER_N_JOBS` to it.

Reply:
Done. I had to split the `make all` and `make install` into two commands, though, as the `install` target does not have any prerequisites in the makefile so with `-j` it would be executed before everything has been built.

I'll contact the upstream author and propose some makefile changes to fix this and the other issues, so we can simplify this again in a future update.